wpeverest/everest-forms
# EVF-580 Feature - Custom CSS and JS for form

### All Submissions:






### Changes proposed in this Pull Request:

This PR introduces a feature to add custom styles( CSS ) and scripts( JS ) to the frontend page where the form is loaded.

### How to test the changes in this Pull Request:

1. Make sure the newly added **Custom CSS and JS** setting menu is added in form builder settings.
2. Verify all the features are working as expected.
3. Try enabling/disabling the css and js. Try different styling and scripts.

### Types of changes:




### Other information:




### Changelog entry

> EVF-580 Feature - Custom CSS and JS for form
